purge fix

czechbol opened the pull request:
fixed a couple of issues. Needs sk.popie

sinus-x commented on purge/module.py:
```diff
@@ -32,21 +31,80 @@ async def purge(self, ctx: commands.Context, num_to_delete: int = None):
+
+            embed = utils.discord.create_embed(
+                author=ctx.author,
+                title="Confirm delete.",
```
Let's make this translatable, too.

sinus-x commented on purge/module.py:
```diff
@@ -32,21 +31,80 @@ async def purge(self, ctx: commands.Context, num_to_delete: int = None):
+            if count > 10:
+                embed = utils.discord.create_embed(
+                    author=ctx.author,
+                    title="Confirm delete.",
```
ditto

czechbol commented:
> The confirmation dialog bumped it up a lot. But that's fine, as this is the only functionality in this module. Better safe than sorry.

Addressed comments, added sk translations

sinus-x reviewed (approved):
LGTM now
